.github/workflows/behavioral-test.lock.yml

```
@@ -0,0 +1,20 @@
+# This file is auto-generated by: gh aw compile behavioral-test.md
+# Do not edit manually. Re-run compile after any frontmatter changes.
+# Generated: placeholder (compile required before first run)
+
+name: "Forge Workflow Behavioral Test"
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)